freqtrade/freqtradebot.py

```diff
@@ -348,26 +347,28 @@ def enter_positions(self) -> int:
         if not whitelist:
             logger.info("Active pair whitelist is empty.")
             return trades_created
-        # Remove pairs for currently opened trades from the whitelist
-        for trade in Trade.get_open_trades():
-            if trade.pair in whitelist:
-                whitelist.remove(trade.pair)
-                logger.debug('Ignoring %s in pair whitelist', trade.pair)
-
-        if not whitelist:
-            logger.info("No currency pair in active pair whitelist, "
-                        "but checking to sell open trades.")
-            return trades_created
-        # Create entity and execute trade for each pair from whitelist
-        for pair in whitelist:
-            try:
-                trades_created += self.create_trade(pair)
-            except DependencyException as exception:
-                logger.warning('Unable to create trade for %s: %s', pair, exception)
+        else:
```

Comment:
`else` unnecessary after an `if` that contains a return. ("Flat is better than nested" - [Zen](https://www.python.org/dev/peps/pep-0020/))